R/utils-shared.R

```diff
@@ -219,5 +219,5 @@ isEmptyOrBlank <- function(val = NULL, listObjects = NULL, objectName = NULL){
 isEmptyVar <- function(variable){
   all(!is.null(variable), 
       nrow(variable) != 0 || is.null(nrow(variable)), 
-      length(variable$time[!is.na(variable$time)]) != 0 & length(variable$value[!is.na(variable$value)]) != 0)
+      length(variable$time[!is.na(variable$time)]) != 0 & length(variable$value) != 0)
```

Comment:
So, I am suggesting deleting the check for `value` and only keeping `length(variable$time[!is.na(variable$time)]) != 0`